Tools/Scripts/webkitpy/common/version_name_map.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, platform=None):
             'ios': self._automap_to_major_version('iOS', minimum=Version(10), maximum=Version(17)),
             'tvos': self._automap_to_major_version('tvOS', minimum=Version(10), maximum=Version(17)),
             'watchos': self._automap_to_major_version('watchOS', minimum=Version(1), maximum=Version(10)),
+            'visionos': {'visionOS 1': Version(1)},
```

Comment:
I know this is redundant here, but can we be consistent with the above and just use:
```suggestion
            'visionos': self._automap_to_major_version('visionOS', minimum=Version(1), maximum=Version(1)),
```

Reply:
We actually cannot because `self._automap_to_major_version` doesn't work if the two versions match.

Reply:
https://github.com/WebKit/WebKit/pull/23775 suggests otherwise 🙃